app/javascript/src/NewApplication/index.js

```diff
@@ -0,0 +1,10 @@
+// @flow
+
+export * from './components/BuyerSelect'
+export * from './components/ServicePlanSelect'
+export * from './components/ProductSelect'
+export * from './components/ApplicationPlanSelect'
+// $FlowFixMe $FlowError[cannot-resolve-module] failing because file name contains module's name NewApplication
```

Comment:
Will https://github.com/3scale/porta/pull/2362 fix all this $FlowFixMe issues?
If true, we should first merge that, and rebase here